fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogBottomSheet.kt

```diff
@@ -0,0 +1,196 @@
+ */
+@Composable
+fun TranslationOptionsDialogBottomSheet(
+    translationOptionsList: List<TranslationSwitchItem>,
+    onBackClicked: () -> Unit,
+    onTranslationSettingsClicked: () -> Unit,
+    aboutTranslationClicked: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .background(
+                color = FirefoxTheme.colors.layer2,
+                shape = RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp),
+            )
+            .nestedScroll(rememberNestedScrollInteropConnection()),
+    ) {
+        TranslationOptionsDialogHeader(onBackClicked)
+
+        LazyColumn {
+            items(translationOptionsList) { item: TranslationSwitchItem ->
+                SwitchWithLabel(
+                    checked = item.isChecked,
+                    onCheckedChange = item.onStateChange,
+                    label = item.textLabel,
+                    modifier = Modifier
+                        .padding(start = 72.dp, end = 16.dp),
+                )
+
+                if (item.hasDivider) {
+                    Divider(Modifier.padding(top = 4.dp, bottom = 4.dp))
+                }
+            }
+
+            item {
+                TextListItem(
+                    label = stringResource(id = R.string.translation_option_bottom_sheet_translation_settings),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(start = 56.dp),
+                    onClick = { onTranslationSettingsClicked() },
+                )
+            }
+
+            item {
+                TextListItem(
+                    label = stringResource(
+                        id = R.string.translation_option_bottom_sheet_about_translations,
+                        formatArgs = arrayOf(stringResource(R.string.app_name)),
+                    ),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(start = 56.dp),
+                    onClick = { aboutTranslationClicked() },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun TranslationOptionsDialogHeader(
+    onBackClicked: () -> Unit,
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .padding(end = 16.dp, start = 16.dp)
+            .defaultMinSize(minHeight = 56.dp),
+    ) {
+        IconButton(
+            onClick = { onBackClicked() },
+            modifier = Modifier
+                .size(24.dp),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.mozac_ic_back_24),
+                contentDescription = stringResource(R.string.etp_back_button_content_description),
+                tint = FirefoxTheme.colors.iconPrimary,
+            )
+        }
+
+        Spacer(modifier = Modifier.width(32.dp))
+
+        Text(
+            text = stringResource(id = R.string.translation_option_bottom_sheet_title),
```

Comment:
Could you add: 
`modifier = Modifier.semantics { heading() }`

(More info below.)

Reply:
Done

Reply:
Done